# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/parent/moz.build
# Commit: e4b3f3d8ba5f
# Full Hash: e4b3f3d8ba5fd951bf79a2014dcee109255c865f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Rename POriginPrivateFileSystem to PFileSystemManager; r=dom-storage-reviewers,jesup
#   
#   The child actor will be held by FileSystemManager on the child side and the
#   parent actor already holds FileSystemDataManager, so it makes sense to rename
#   the protocol to PFileSystemManager.
# ==============================================================================

diff -r 3b0da4b4bb83 -r e4b3f3d8ba5f dom/fs/parent/moz.build
--- a/dom/fs/parent/moz.build	Wed Aug 31 09:34:48 2022 +0000
+++ b/dom/fs/parent/moz.build	Wed Aug 31 09:34:48 2022 +0000
@@ -5,14 +5,14 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 EXPORTS.mozilla.dom += [
+    "FileSystemManagerParent.h",
     "FileSystemManagerParentFactory.h",
-    "OriginPrivateFileSystemParent.h",
 ]
 
 UNIFIED_SOURCES += [
     "FileSystemHashSource.cpp",
+    "FileSystemManagerParent.cpp",
     "FileSystemManagerParentFactory.cpp",
-    "OriginPrivateFileSystemParent.cpp",
 ]
 
 FINAL_LIBRARY = "xul"